docs/Zendesk-batchsource.md

```diff
@@ -48,19 +47,19 @@ the specified date. The date must be provided in the date format:
 |                                  | YYYY-MM-DDThh:mm:ss-hh:mm | 1999-01-01T23:01:01-08:00 |
 |                                  | YYYY-MM-DDThh:mm:ssZ      | 1999-01-01T23:01:01Z      |
 
-Specifying this along with `Start Date` allows reading data modified within a specific time window.
+If you enter an End Date and Start Date, the data modified within a specific time window.
 If no value is provided, no upper bound is applied.
 
-**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records which have Zendesk score
-equal to the specified score.
+**Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records that have a Zendesk score
+equal to the specified score. Only applicable for the Satisfaction Rating object
 
 ### Advanced
 
-**Max Retry Count:** Maximum number of retry attempts.
+**Max Retry Count:** Maximum number of retry attempts.Default is 20.
 
-**Connect Timeout:** Maximum time in seconds connection initialization can take.
+**Connect Timeout:** Maximum time in seconds connection initialization. Default is 300.
 
-**Read Timeout:** Maximum time in seconds fetching data from the server can take.
+**Read Timeout:** Maximum time in seconds fetching data from the server. Default is 300.
```

Comment:
This is the wording in our Google doc: Maximum time in seconds to fetch data from the server.

Reply:
updated